VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/services/video-call.service.ts

```diff
@@ -77,7 +77,7 @@ export class VideoCallService {
         };
     }
 
-    private async retrievePreferredDevices() {
+    public async retrievePreferredDevices() {
```

Comment:
You have made this public but it doesn't look like this method is being called outside of this class. 
![image](https://user-images.githubusercontent.com/41630528/96244586-2ed6a380-0f9e-11eb-9bda-b3d2c0a09d56.png)

Reply:
replace to private